js/main.js

```diff
@@ -1,25 +1,118 @@
-const $imgLoad = document.querySelector('#user-photo-url');
-const $image = document.querySelector('#imgshow');
-$imgLoad.addEventListener('input', function (event) {
+const div = document.querySelector('.all-entries');
+const photoURL = document.getElementById('image-url');
+const entriesPage = document.getElementById('entries-page');
 
-  $image.setAttribute('src', event.target.value);
+photoURL.addEventListener('input', function (event) {
+  const img = document.querySelector('img');
+  img.src = event.target.value;
 });
-
-const $page = document.querySelector('#information');
-
-function pageSubmit(event) {
+const submitForm = document.querySelector('form');
+submitForm.addEventListener('submit', function (event) {
   event.preventDefault();
-  const entry = {
-    entryId: data.nextEntryId,
-    title: event.target.elements.title.value,
-    photoUrl: event.target.elements.url.value,
-    notes: event.target.elements.notes.value
+  const title = submitForm.elements.title.value;
+  const image = submitForm.elements.image.value;
+  const notes = submitForm.elements.notes.value;
+  const newObject = {
+    title,
+    image,
+    notes,
+    entryId: data.nextEntryId
   };
-
-  $page.reset();
+  data.entries.unshift(newObject);
   data.nextEntryId++;
-  data.entries.unshift(entry);
-  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  const img = document.querySelector('img');
+  img.src = './images/placeholder-image-square.jpg';
+  submitForm.reset();
+
+  const newEntry = renderEntry(newObject);
+  div.prepend(newEntry);
+  viewSwap('entries');
+});
+
+function renderEntry(entry) {
+
+  const ul = document.createElement('ul');
+  ul.setAttribute('class', 'is-flex');
```

Comment:
You should not be creating a `ul` inside of here. The outermost element should be an `li`